plugins/destination/postgresql/client/client_test.go

```diff
@@ -24,3 +32,44 @@ func TestPgPlugin(t *testing.T) {
 		},
 		destination.PluginTestSuiteTests{})
 }
+
+func TestPgPluginPrimaryKeyRename(t *testing.T) {
+	tableName := fmt.Sprintf("cq_test_pk_rename_%d", rand.Intn(100))
```

Comment:
This is so we don't need to drop the table before each test

Reply:
Might work or not work depending on random seed :)

Reply:
Yeah, this is mostly for local dev, the CI always starts with a new DB